src/test/java/org/ecommerce/services/jpa/impl/OrderJpaServiceImplTest.java

```diff
@@ -216,6 +231,70 @@ void findAll() {
         );
     }
 
+    @DisplayName("Assess the findAll metadata")
+    @Test
+    void findAllAssesPagination() {
+        final List<Order> mockedList = List.of(order, order, order);
+        when(page.getContent()).thenReturn(mockedList);
+
+        when(orderDTOMapper.apply(any(Order.class))).thenReturn(orderDTO);
+        when(orderJpaRepository.findAll(any(Specification.class), any(Pageable.class)))
```

Comment:
Please create a separate unit test to actually test `orderJpaRepository.findAll(specification, pageable)`

Reply:
Here are the tests without mocking the repository outputs https://github.com/algutierrez-griddynamics/e-commerce/pull/58/commits/79a3d48ee302b42065b5fbf518afe2a6baf52b2e 
<img width="475" alt="image" src="https://github.com/user-attachments/assets/6507e081-d7b0-47a7-9e55-fb1ba6ac04fa">